.github/workflows/release-shared.yml

```
@@ -0,0 +1,439 @@
+              echo "::error::Package $pkg@$VERSION not found on NPM"
+              exit 1
+            fi
+
+            echo "✅ $pkg@$VERSION verified"
+          done
+
+          echo "All packages verified successfully"
+          echo "::endgroup::"
+
+      - name: 📋 Log changes (dry run only)
+        if: ${{ inputs.dryRun }}
+        env:
+          RELEASE_BRANCH: ${{ needs.calculate.outputs.target-branch }}
+        run: |
+          git diff "origin/${RELEASE_BRANCH}"
+          echo "DRY RUN: No changes were made."
+
+      - name: 🚀 Push Tags and Commits
+        id: push-tags
+        shell: bash
+        if: ${{ !inputs.dryRun }}
+        env:
+          NEW_LTS_BRANCH: ${{ needs.calculate.outputs.next-lts-branch }}
+        run: |
+          echo "Git log:"
+          git log --oneline -n 5
+
+          # Get the most recent tag (the one we just created)
+          RELEASE_TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
+          echo "Release tag: $RELEASE_TAG"
+
+          echo "Pushing tags and commits..."
+          # Push tags and commits without running the pre-push hook
+          git push --tags --no-verify && git push --no-verify
+
+          # Pass the release tag to the next step
+          echo "RELEASE_TAG=$RELEASE_TAG" >> $GITHUB_OUTPUT
+
+          # Push the maintenance branch if it exists locally
+          if git rev-parse --verify --quiet "${NEW_LTS_BRANCH}" >/dev/null; then
+            echo "Pushing the maintenance branch: ${NEW_LTS_BRANCH}..."
+            git push --no-verify --set-upstream origin "${NEW_LTS_BRANCH}"
+            echo "✅ Successfully pushed maintenance branch ${NEW_LTS_BRANCH}"
+          fi
+
+      # Create appropriate milestones based on the release type
+      - name: 🏷️ Create GitHub Milestones
+        # Only run in the official repository and not during dry runs
+        # This prevents milestone creation in forks (which would fail due to permissions)
+        # and ensures no milestones are created during dry runs (which should be read-only)
+        if: ${{ github.repository == 'webdriverio-community/wdio-electron-service' && !inputs.dryRun }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.github-token }}
+          RELEASE_TYPE: ${{ inputs.releaseVersion }}
+        run: |
+          echo "::group::Creating GitHub milestones"
+          pnpx tsx scripts/create-milestones.ts
+          echo "::endgroup::"
+
+      - name: 📝 Create GitHub Release
+        if: ${{ github.repository == 'webdriverio-community/wdio-electron-service' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          draft: true # Create as draft for manual review
+          generate_release_notes: true
+          prerelease: ${{ contains(inputs.releaseVersion, 'pre') }}
+          repository: webdriverio-community/wdio-electron-service
+          tag_name: ${{ steps.push-tags.outputs.RELEASE_TAG }}
+          token: $GITHUB_TOKEN
```

More appropriately, the Token itself should be passed as a string, which is easier to understand, so a change is suggested.
```suggestion
          token: ${{ secrets.github-token }}
```

---

I think this has been resolved in the split